src/app/view/perspectives/chart/chart-visualization/chart-visualization.component.scss

```diff
@@ -0,0 +1,7 @@
+:host ::ng-deep .scatterlayer .trace:last-child .points path {
+  pointer-events: all;
+}
+
+:host ::ng-deep .js-plotly-plot .plotly .modebar {
```

Comment:
I can see that the template of this component contains just a single `<div>` element which is used by Plotly to render the graph. Because of this we can use `ViewEncapsulation.None` for this component in order to avoid using `::ng-deep` here. What do you think?

Reply:
That's good idea